modin/experimental/cloud/rayscale.py

```diff
@@ -141,18 +141,23 @@ def __make_config(self):
 
         return _bootstrap_config(config)
 
-    @classmethod
-    def _conda_requirements(cls):
+    def _conda_requirements(self):
         reqs = []
 
-        reqs.append(f"python=={cls._get_python_version()}")
+        reqs.append(f"python=={self._get_python_version()}")
 
-        return reqs
+        if self.add_conda_packages:
+            reqs.extend(self.add_conda_packages)
 
-    @classmethod
-    def _update_conda_requirements(cls, setup_commands: str):
+        # this is needed, for example, for dependencies that
+        # looks like: "scikit-learn>=0.23"
+        reqs_with_quotes = [f'"{req}"' for req in reqs]
```

Comment:
I still think this should be handled by `shlex.quote()` instead of re-inventing the wheel

Reply:
Ok, use shlex in the case